homework_7_261122/src/triangles/Triangle.java

```diff
@@ -0,0 +1,51 @@
+package triangles;
+
+public class Triangle {
```

Comment:
you need to add constuctor and call verifyPythagoras() method, before creation

Reply:
Fixed. Created constuctor with all sides and a condition to test the Pythagorean theorem.